.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Feature Request
+    url: https://github.com/Kong/kong/discussions/categories/ideas
```

Comment:
Question: do we need to enable `Discussions` for this repo? My concern is that if we do, users will sometimes discuss Gateway-related(not GUI-related) topics here.

Reply:
I don't think there's any problem; If it is really to discuss kong related issues, can we push it to the Kong side?
What do you think?

Reply:
That's true. However, we may utilize [GitHub's transferring discussions feature](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#transferring-a-discussion) to move off-topic discussions to the right place.

Reply:
Makes sense. Didn't know discussions can be transferred.

Reply:
I enabled `Discussions` for this repo and updated code accordingly.

Reply:
Sounds good.